cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -328,6 +339,33 @@ protected void shutDown() throws Exception {
     cliService.stop();
   }
 
+  private void setupSparkConf() {
+    // Copy over hadoop configuration as spark properties since we don't localize hadoop conf dirs due to CDAP-5019
+    for (Map.Entry<String, String> entry : hConf) {
+      sparkConf.put("spark.hadoop." + entry.getKey(), entry.getValue());
```

Comment:
You have to do `hConf.get(entry.getKey())`, otherwise the value won't get resolved and will be causing errors in the remote driver/executor processes (e.g. in hdp, there can be ${hdp.version} in the hConf value, which it should get resolved with the system property and likely that property is not set on the driver/executor processors).